src/transformers/feature_extraction_utils.py

```diff
@@ -354,6 +354,8 @@ def save_pretrained(self, save_directory: Union[str, os.PathLike], push_to_hub:
             url = self._push_to_hub(repo, commit_message=commit_message)
             logger.info(f"Feature extractor pushed to the hub in this commit: {url}")
 
+        return [output_feature_extractor_file]
```

Comment:
@sgugger - analogue to tokenizers, let's also output a list of saved files for the feature extractors no? Don't think this can break anything

Reply:
Works for me!